HomeAssistant/Classes/Components/Climate.swift

```diff
@@ -56,7 +56,7 @@ class Climate: Entity {
     }
 
     func TurnFanOn() {
-        _ = HomeAssistantAPI.sharedInstance.CallService(domain: "climate",
+        _ = HomeAssistantAPI.authenticatedAPI()?.CallService(domain: "climate",
```

Comment:
What happens if authenticatedAPI returns nil?

Reply:
fails silently. I think that's ok as other parts of the app should handle re-directing the user to authenticate if we are in that state.